firmware/controllers/trigger/trigger_decoder.cpp

```diff
@@ -389,7 +404,7 @@ angle_t PrimaryTriggerDecoder::syncEnginePhase(int divider, int remainder, angle
 	}
 
 	// Allow injection/ignition to happen, we've now fully sync'd the crank based on new cam information
-	m_hasSynchronizedPhase = true;
+	engine->engineState.isSynchronizedPhase = true;
```

Comment:
wtf? please don't split that state outside of the trigger decoder

Reply:
WTF yourself for "let me just manually copy this boolean from here to there"

https://github.com/rusefi/rusefi/commit/2baae2adc0a44510044cf3d145759877785b5f53